### DISCUSSION TITLE
manually migrate QKSMS SMS/MMS to QUIK help

### DISCUSSION THREAD

**Discussion by bizwog** [octoshrimpy/quik#194] [Q&A]
I intended to move my QKSMS messages from LineageOS to DivestOS but I'm unable to backup the QKSMS with any FDROID apps or SMS Backup & Restore from Google Play. I'm not sure why it's erroring out. 

I'd like to be able to move the QKSMS data to QUIK so I can use the QUIK export function to get the data and manually copy it over to QUIK on DivestOS. I don't have the QKSMS+ version so I cannot use the export feature with QKSMS. 

Please help me move the QKSMS data to QUIK on the LineageOS phone.

**Comment by octoshrimpy** [comment]
I believe you will find help in this issue: https://github.com/octoshrimpy/quik/issues/70.
